reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java

```diff
@@ -485,4 +494,6 @@ public String toString() {
 	};
 
 	static final Logger log = Loggers.getLogger(FluxReceive.class);
+
+	static final Logger logLeakDetection = Loggers.getLogger("reactor.netty.channel.LeakDetection");
```

Comment:
Maybe this could be a log category that remains off by default when "reactor.netty" is on. In the Spring Framework we use an ["_" prefix](https://github.com/spring-projects/spring-framework/blob/31d9f31806fec88b268e1ad97cfa2db428d3f734/spring-core/src/main/java/org/springframework/core/log/LogDelegateFactory.java#L73), one possible idea.

Reply:
fixed